test/unit/util/rational_white.h

```diff
@@ -35,7 +35,7 @@ class RationalWhite : public CxxTest::TestSuite {
 
   void testCompareAgainstZero(){
     Rational q(0);
-    TS_ASSERT_THROWS_NOTHING(q == 0;);
+    TS_ASSERT_THROWS_NOTHING(q = 0;);
```

Comment:
Why did you change this?

Reply:
Huh, good catch. Not sure how that happened.